[ADD] Update MX loc info

mial-odoo opened the pull request:
Since v16 FEAV was doing the doc, he left the company and left the changes unfinished, I'll be taking over and get working on the V18 doc since we have a huge gap between the current features and what is on the public doc.

-MIAL PO of MX office and MX Loc

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/12323.png)](https://mergebot.odoo.com/odoo/documentation/pull/12323)

mial-odoo commented:
Reviewed up to line 777.

mial-odoo commented:
This has been sitting idle for months and now we are getting complaints, might as well release what we have, it even outdated by now.... odoo moves too fast.

mial-odoo commented:
NGL, this needs serious restructuring, will do that for v19 doc since we need to add tons of features now.

mial-odoo commented:
@dade-odoo I did not want to fix anything yet because it was still missing several pics and fixes but it doesn't look like there will be more time for now to fix it, so I added some additional fixes, added the missing pictures and deleted a few pics to be able to merge.
I can't quash commits since I don't use VScode but it is ready :)
